plugins/techdocs-backend/README.md

```diff
@@ -6,16 +6,39 @@ This is the backend part of the techdocs plugin.
 
 This backend plugin can be started in a standalone mode from directly in this package
 with `yarn start`. However, it will have limited functionality and that process is
-most convenient when developing the catalog backend plugin itself.
+most convenient when developing the techdocs backend plugin itself.
 
-To evaluate the catalog and have a greater amount of functionality available, instead do
+To evaluate techdocs and have a greater amount of functionality available, instead do
 
 ```bash
 # in one terminal window, run this from from the very root of the Backstage project
 cd packages/backend
 yarn start
+
+# open another terminal window, and run the following from the very root of the Backstage project
+yarn lerna run mock-data
 ```
 
+## What techdocs-backend does
+
+This plugin is the backend part of the techdocs plugin. It provides building and serving of your docs without having to use another service and hosting provider. To use it set your techdocs storageUrl in your `app-config.yml` to `http://localhost:7000/techdocs/static/docs`.
```

Comment:
I think we should give the whole snippet in the app-config.yml here to avoid confusion on where to set the storageUrl